open_academy/views/course_views.xml

```diff
@@ -9,6 +9,8 @@
                     <group>
                         <group>
                             <field name="name"/>
+                            <field name="responsible_user_id"/>
+                            <field name="session_id"/>
```

Comment:
You changed this field from `session_id` to `session_ids`, please whenever you get a review test again all your changes

We do not want a server error in production

Reply:
Take it for grantend